packages/application-config/src/process-config.ts

```diff
@@ -28,17 +32,23 @@ type ProcessConfigOptions = Partial<LoadingConfigOptions> & {
 const developmentPort = 3001;
 const developmentAppUrl = `http://localhost:${developmentPort}`;
 
+const getConfigurationType = (configFileName: string): ConfigType => {
```

Comment:
As we will need to process some things differently, we need to distinguish between Custom Apps and Custom Views configuration files. We do it by inspecting the config file name as they are predefined.